frontend/app/work_packages/controllers/work-package-new-controller.js

```diff
@@ -120,6 +122,7 @@ module.exports = function(
 
     EditableFieldsState.save(function() {
       $rootScope.$emit('workPackagesRefreshRequired');
+      NotificationsService.addSuccess(I18n.t('js.notice_successful_create'));
```

Comment:
This will never be called currently since the callback in save has been removed: 
https://github.com/opf/openproject/blob/dev/frontend/app/components/inplace-edit/services/editable-fields-state.service.js#L51-L65